src/ray/object_manager/object_directory.h

```diff
@@ -174,6 +184,12 @@ class ObjectDirectory : public ObjectDirectoryInterface {
     std::unordered_map<UniqueID, OnLocationsFound> callbacks;
     /// The current set of known locations of this object.
     std::unordered_set<ClientID> current_object_locations;
+    /// Specify whether the object is inlined.
```

Comment:
Add a description of what inlined means. Also specify what the contents of `current_object_locations` will be if this is true/false.